gdc_filtration_tools/__main__.py

```diff
@@ -44,6 +44,7 @@ def main(args: List[str] = None) -> None:
     defopt.run(
         funcs,
         argv=args if args is not None else sys.argv[1:],
+        version=True,
```

Comment:
oh just looked up how defopt handles this, cool

Reply:
Yeah this is nicer than the underlying argparse setup:

`add_argument('--version', action='version', version='')`

Gotta be so repetitive :(